torch/_inductor/metrics.py

```diff
@@ -7,16 +7,21 @@
 # counters for tracking fusions
 ir_nodes_pre_fusion = 0
 
+# counters for tracking to_dtype inserted
+to_dtype_count = 0
```

Comment:
If it is cpp specific, add "cpp" prefix?

Reply:
Done